docs/typescript/activities.md

```diff
@@ -177,12 +177,12 @@ export async function DynamicWorkflow(activityName, ...args) {
 Type safety is still supported here, but you are encouraged to validate and handle mismatches in Activity names. An invalid Activity name will lead to a `NotFoundError` with a message that looks like:
 
 ```
-ApplicationFailure: Activity function fakeProgresss is not registered on this Worker, available activities: ["fakeProgress"]
+ApplicationFailure: Activity function fakeProgress is not registered on this Worker, available activities: ["fakeProgress"]
 ```
 
 ### Using pure ESM Node Modules
 
-The TypeScript ecosystem is increasingly moving towards publishing ES Modules over CommonJS, for example `node-fetch@3` is ESM while `node-fetch@2` is CJS.
+The JavaScript ecosystem is increasingly moving towards publishing ES Modules over CommonJS, for example `node-fetch@3` is ESM while `node-fetch@2` is CJS.
```

Comment:
This implies the TypeScript ecosystem's move towards publishing ES Modules is its own group decision, which might mislead folks to thinking that's a TypeScript-specific thing.

The TypeScript team has had enough woes convincing functional programmers that TypeScript doesn't force your code to look like Java and wrestling with various module systems support. They probably don't want to have yet another opinion to discuss. 😄